pluggy/callers.py

```diff
@@ -31,12 +31,17 @@ def __init__(self, result, excinfo):
 
     @property
     def excinfo(self):
+        """Get the exception info for this hook call (DEPRECATED in favor of
+        ``get_result()``).
+        """
+        msg = 'Use `get_result()` which raises the underlying exception'
```

Comment:
i believe for clarity of the warning the property should be referenced in the text somehow